agent/windows-setup-agent/SetupWizard.cs

```diff
@@ -238,14 +238,12 @@ private void ConfigureService()
 
 			string serviceUser = txtUser.Text.Trim();
 
-			DirectoryInfo di = new DirectoryInfo(Program.Icinga2InstallDir);
-			DirectorySecurity ds = di.GetAccessControl();
 			FileSystemAccessRule rule = new FileSystemAccessRule(serviceUser,
 				FileSystemRights.Modify,
 				InheritanceFlags.ObjectInherit | InheritanceFlags.ContainerInherit, PropagationFlags.None, AccessControlType.Allow);
 			try {
-				ds.AddAccessRule(rule);
-				di.SetAccessControl(ds);
+				AddAccessRuleToFSTree(rule, Program.Icinga2InstallDir);
```

Comment:
Why is this even done on `Program.Icinga2InstallDir` (which should be `C:\Program Files\ICINGA2`)? I see that this was already done before, but I don't understand why and don't see the need.

Reply:
And I don’t want to touch this running system.